src/main/java/org/eulu/bookshop/model/AppUser.java

```diff
@@ -0,0 +1,27 @@
+package org.eulu.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "app_users")
```

Comment:
```suggestion
@Table(name = "users")
```
Remove it everywhere